tests/foreman/destructive/test_remoteexecution.py

```diff
@@ -58,16 +59,20 @@ def test_negative_run_capsule_upgrade_playbook_on_satellite(target_sat):
                     'whitelist_options': "repositories-validqqate,repositories-setup",
                 },
                 'targeting_type': "static_query",
-                'search_query': f"name = {sat.name}",
+                'search_query': f"name = {target_sat.hostname}",
             }
         )
     assert 'A sub task failed' in error.value.args[0]
     job = target_sat.api.JobInvocation().search(
-        query={'search': f'host={sat.name},status=failed,description="Capsule Upgrade Playbook"'}
+        query={
+            'search': f'host={target_sat.hostname},'
+            'status=failed,description="Capsule Upgrade Playbook"'
+        }
     )[0]
+    host = target_sat.api.Host().search(query={'search': target_sat.hostname})
     response = client.get(
-        f'{target_sat.url}/api/job_invocations/{job.id}/hosts/{sat.id}',
-        auth=(target_sat.username, target_sat.password),
+        f'{target_sat.url}/api/job_invocations/{job.id}/hosts/{host[0].id}',
+        auth=(settings.server.admin_username, settings.server.admin_password),
         verify=False,
```

Comment:
why not implement this endpoint in nailgun and use it here with target_sat? (non-blocking)

Reply:
good idea, I filed https://github.com/SatelliteQE/nailgun/issues/849 to track this